heartbeat/azure-events.in

```diff
@@ -82,9 +82,19 @@ class azHelper:
 		req = urllib2.Request(url, postData)
 		req.add_header("Metadata", "true")
 		req.add_header("User-Agent", USER_AGENT)
-		resp = urllib2.urlopen(req)
-		data = resp.read()
-		ocf.logger.debug("_sendMetadataRequest: response = %s" % data)
+		try:
+			resp = urllib2.urlopen(req)
+		except URLError as e:
+			if hasattr(e, 'reason'):
+				print('We failed to reach a server. Reason: '), e.reason
```

Comment:
You should use e.g. ocf.logger.info() or warn() instead of print().

Also you should make e.reason (and e.code in the other print() statement) part of the string in the function.

Reply:
Thanks for reviewing. Now using ocf.logger.warning() instead or print() (with e.reason and e.code inside the function).